src/core/binding.c

```diff
@@ -955,7 +955,7 @@ QuicBindingProcessStatelessOperation(
             PacketLength - QUIC_STATELESS_RESET_TOKEN_LENGTH,
             SendDatagram->Buffer);
         ResetPacket->IsLongHeader = FALSE;
-        ResetPacket->FixedBit = 1;
+        ResetPacket->FixedBit = 1; // TODO : Find a way to check connection settings and PeerTPs.
```

Comment:
There is no connection state in this case. That's the point of stateless reset. Just always assume grease was disabled.

Reply:
Alright!